requirements.txt

```diff
@@ -3,4 +3,4 @@ pipdeptree==0.13.2
 QuantStats==0.0.25
 tqdm==4.43.0
 wheel==0.34.2
-xlrd==1.2.0
+xlrd==1.2.0
```

Comment:
I expected to see here the same (or more) packages than `requirements-base` has,  with their versions.